chat-commands.js

```diff
@@ -128,6 +128,34 @@ exports.commands = {
 			'How many digits of pi do YOU know? Test it out <a href="http://guangcongluo.com/mempi/">here</a>!');
 	},
 
+	code: function (target, room, user) {
+		if (!target) return this.parse('/help code');
+		if (!this.canTalk()) return;
+		const separator = '\n';
+		if (target.includes(separator)) {
+			const params = target.split(separator);
+			let output = [];
+			for (let i = 0; i < params.length; i++) {
+				output.push(Chat.escapeHTML(params[i]));
+			}
+			let code = `<code style="white-space: pre-wrap">${output.join('<br />')}</code>`;
```

Comment:
Didn't look weird to me; it actually looked like more-realistic sized spaces than it was before.

Preview: 
<img src="https://image.prntscr.com/image/KLgWychBRBSybo8S_knAlw.png">